language/tools/utils/src/inhabitation/inhabit.rs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use vm::file_format::{
+    AddressPoolIndex, ByteArrayPoolIndex, Bytecode, CompiledModuleMut, LocalsSignatureIndex,
+    SignatureToken, StructDefinitionIndex, StructFieldInformation, TableIndex, UserStringIndex,
+};
+
+// Generate a sequence of instructions that inhabits the value of `token`
```

Comment:
I believe the more precise description of the behavior of this function (based on our offline discussion) is as follows:

Generate a sequence of instructions whose overall effect is to push a single value of type `token` on the stack, specifically without consuming any values that existed on the stack prior to the execution of the instruction sequence.

If you agree with my description, please incorporate it in the comment.